enhancements/update/distribute-secondary-metadata-as-container-image.md

```diff
@@ -0,0 +1,184 @@
+---
+title: cincinnati-graph-data-package-distribute-and-consume-in-a-container-image-repository
+authors:
+  - "@steveeJ"
+reviewers:
+  - @openshift/openshift-team-cincinnati-maintainers
+  - @openshift/openshift-team-cincinnati
+  - @openshift/team-cincinnati-operator
```

Comment:
I do not think github sends any kind communication to the alias we have mentioned here. Wondering if we mention a single person's alias, the github notification works or not.

Reply:
I don't think so either. Once all of the reviewer's comments are resolved I'll ping the approvers in the PR description.